AIPscan/Aggregator/templates/storage_service.html

```diff
@@ -49,7 +49,16 @@
       <tr>
         <td>{{ mets_fetch_job.download_start }}</td>
         <td>{% if mets_fetch_job.download_end %} {{ mets_fetch_job.download_end - mets_fetch_job.download_start }} {% endif %}</td>
-        <td>{{ mets_fetch_job.total_packages }}</td>
+        <td>{{ mets_fetch_job.total_packages }} packages
+          <i class="fa fa-info-circle fa-lg" id="{{ mets_fetch_job.id }}" style="margin-left: 5px;"></i><br/>
+          <div id="stats-{{ mets_fetch_job.id }}" class="stats" style="font-size:80%; color: #999;">
```

Comment:
I'd recommend using Bootstrap tags and classes instead of a custom `style` tag to manage the styles. So for instance,  inside this `<div>` you could do something like
```
<span class="text-muted">
  <small>
    {{ mets_fetch_job.total_aips }} AIPs<br>
    {{ mets_fetch_job.total_sips }} SIPs<br>
    {{ mets_fetch_job.total_dips }} DIPs<br>
    {{ mets_fetch_job.total_replicas }} replicated AIPs<br>
    {{ mets_fetch_job.total_replicas }} deleted AIPs
  </small>
</span>
```
We do that elsewhere in the code base as well, e.g.: https://github.com/artefactual-labs/AIPscan/blob/main/AIPscan/Reporter/templates/report_largest_files.html#L74. The main advantage this change would give is if we want to change what our `text-muted` style looks like, we do so in one place and have it apply to the whole project (and full transparency, on my mind because I think the low contrast might be a bit of an accessibility issue, and so we might want to find a less-muted shade of grey).

Reply:
Yes, custom inline CSS is a bad temp hack. Thanks for this cleanup suggestion. Done.